scripts/upgrade.sh

```diff
@@ -148,4 +148,11 @@ function main() {
     popd_install_directory
 }
 
-main "$@"
+# tee logs into /var/log/kurl/upgrade-<date>.log and stdout
+mkdir -p /var/log/kurl
+LOGFILE="/var/log/kurl/upgrade-$(date +"%Y-%m-%dT%H-%M-%S").log"
+tasks "$@" 2>&1 | tee -a $LOGFILE
```

Comment:
```suggestion
main "$@" 2>&1 | tee -a $LOGFILE
```

Reply:
this does not need append does it?

```suggestion
main "$@" 2>&1 | tee $LOGFILE
```